### ISSUE TITLE
Question about Focal Loss

### ISSUE THREAD

**Issue by HerrYu123** [sunggukcha/deeplabs#10]
$\alpha_t$ in the focal loss is  predefined in the ‘weight’ of torch.nn.CrossEntropyLoss. So I think the 'alpha' in the ./utils/loss/FocalLoss doesn't need to be added again. Did I get it wrong?
